draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -265,17 +265,78 @@ fall outside the scope of these guidelines are certainly possible;
 these guidelines should not be considered as an all-encompassing
 check-list.
 
-(0)
-: Differences with Congestion Control Principles {{!RFC2914}}
+When considering a new congestion control proposal, the community MUST
+consider the following criteria. These criteria will be evaluated in various
+domains (see {{general-use}} and {{special-cases}}).
 
-: Proposed congestion control mechanisms should include a clear
-explanation of the deviations from {{!RFC2914}}.
+## Single Algorithm Behavior
+
+The following criteria evaluate the proposed algorithm when one or more
+flows using that algorithm share a bottleneck link, with no other algorithms
+operating.
+
+### Protection Against Congestion Collapse
+
+The alternate congestion control mechanism should either stop
+sending when the packet drop rate exceeds some threshold
+{{?RFC3714}}, or should include some notion of "full backoff".  For
+"full backoff", at some point the algorithm would reduce the
+sending rate to one packet per round-trip time and then
+exponentially backoff the time between single packet
+transmissions if congestion persists.  Exactly when either "full
+backoff" or a pause in sending comes into play will be
+algorithm-specific.  However, as discussed in {{!RFC2914}}, this
+requirement is crucial to protect the network in times of extreme
+congestion.
+
+If "full backoff" is used, this bullet does not require that the
+full backoff mechanism must be identical to that of TCP
```

Comment:
/the result of full backoff/ ... or something - rather than the internal algorithm detail?